OWNERS_ALIASES

```diff
@@ -83,10 +87,12 @@ aliases:
   azure-approvers:
     - fabianofranz
```

Comment:
fabiano could probably be removed here as well, although he is always welcome :)